src/main/java/guessingnumber/Application.java

```diff
@@ -1,7 +1,18 @@
 package guessingnumber;
 
+import java.io.IOException;
+
 public class Application {
-    public static void main(String[] args) {
-        // TODO: 프로그램 구현
+    public static void main(String[] args) throws IOException {
+        boolean flag = false; // 재시작 플래그
```

Comment:
재시작 플래그라는 주석을 달아야만 이해할 수 있는 flag변수.. 정말 좋은 코드일까요?

변수명에 의미를 담아보는 건 어떨까요?
```suggestion
        boolean restartFlag = false; // 재시작 플래그
```